sleuth/sleuth/enrich_tls.py

```diff
@@ -203,20 +268,37 @@ def enrich_tls(flow, kwargs):
         else:
             scs = None
 
+
         if 'server_cert' in tls:
             certs = list()
             for x in tls['server_cert']:
                 tmp = dict()
-                tmp['signature_algorithm'] = x['signature_algorithm']
-                tmp['signature_key_size'] = x['signature_key_size']
+                tmp['sig_alg'] = x['signature_algorithm']
+                tmp['sig_key_size'] = x['signature_key_size']
+                tmp['issuer'] = x['issuer']
                 certs.append(tmp)
         else:
             certs = None
-
+        
         seclevel_policy = Policy(kwargs["policy_file"], kwargs["failure_threshold"])
 
+        certs_seclevel, certs_concerns = get_certs_seclevel(seclevel_policy, certs, kwargs["unknowns"])
+        tls_seclevel, tls_concerns = get_tls_seclevel(seclevel_policy, scs, client_key_length, kwargs["unknowns"])
+
         tls_sec_info = {}
-        tls_sec_info["seclevel"] = tls_seclevel(seclevel_policy, kwargs["unknowns"], scs, client_key_length, certs)
+        concerns = certs_concerns + tls_concerns
+        seclevels = [ certs_seclevel, tls_seclevel ]
+
+        classification = seclevel_policy.seclevel(min(x for x in seclevels if x is not None))
+
+        if not concerns:
+            tls_sec_info["seclevel"] = classification
+        else:
+            # remove any duplicate entries... usually from cert audit
+            concerns = list(set(concerns))
```

Comment:
if there's a list of certificates, would it make sense to embed the certificate of the chain so that user can see which ones have the same concern?